contracts/staking/StakingPool.sol

```diff
@@ -71,10 +71,15 @@ contract StakingPool is
         _;
     }
 
+    modifier minTotalPoolStakeMet() {
```

Comment:
suggestion: rename to `hasMinimumStake`; neater language.

_You are checking the Pool has the minimum stake_